policy-service/src/contract-tests/java/uk/gov/gchq/palisade/contract/policy/rest/HealthActuatorContractTest.java

```diff
@@ -37,9 +38,10 @@
 @SpringBootTest(
         classes = PolicyApplication.class,
         webEnvironment = WebEnvironment.RANDOM_PORT,
-        properties = {"spring.autoconfigure.exclude=org.springframework.boot.autoconfigure.data.redis.RedisAutoConfiguration"}
+        properties = {"management.health.kafka.enabled=false", "spring.autoconfigure.exclude=org.springframework.boot.autoconfigure.data.redis.RedisAutoConfiguration"}
 )
 @ActiveProfiles("akka-test")
+@TestPropertySource(properties = "management.health.kafka.enabled=false")
```

Comment:
Why is this value in 2 places?

Reply:
Too quick with the copy and paste.  Have removed the whole line starting with @TestPropertySource... plus the second property in the @SpringBootTest properties lines starting with "spring.autoconfigure..."